docs/rules/indent.md

```diff
@@ -0,0 +1,82 @@
+# This option sets a specific tab width for your code (indent)
+
+This option validates a specific tab width for your code.
+
+## Rule Details
+
+This rule is aimed at increasing code readability by enforcing consistent indentation style. The default style is `4 spaces`.
```

Comment:
Let's remove "readability" from this. It's really just about ensuring consistent indentation.